docs/pages/components/button.md

```diff
@@ -8,116 +8,67 @@ permalink: components/button.html
+
+- **Default Button**
+- **Emphasized Button** Used for primary action
+- **Ghost Button** It is used for secondary actions or primary button in case there is already a page primary button
+- **Standard Button** Neutral or informative (secondary) actions
+- **Positive Button** Used for positive (secondary) actions
+- **Negative Button** Used for negative (secondary) actions
+- **Attention Button**
+- **Transparent Button** Used for secondary or negative path actions
 
 {% capture button %}
-<button class="fd-button">Action Button</button>
+<button class="fd-button">Default Button</button>
+<button class="fd-button fd-button--emphasized">Emphasized Button</button>
+<button class="fd-button fd-button--ghost">Ghost Button</button>
 <button class="fd-button fd-button--standard">Standard Button</button>
 <button class="fd-button fd-button--positive">Positive Button</button>
 <button class="fd-button fd-button--negative">Negative Button</button>
+<button class="fd-button fd-button--attention">Attention Button</button>
+<button class="fd-button fd-button--transparent">Transparent Button</button>
 {% endcapture %}
 {% include display-component.html component=button %}
 
+<br>
 
-## Button Sizes
-There are two sizes.
+## Button Types
 
-> The compact size is only used on desktop and it is full size when used on a touch device.
-
-{% capture button %}
-<button class="fd-button">Default</button>
-<button class="fd-button fd-button--compact">Compact</button>
-{% endcapture %}
-{% include display-component.html component=button %}
+### Simple Button
 
+It is recommended to use simple button for specific actions, such as _Create, Edit, Save, Approve, Reject, Accept, Decline, OK, Cancel_
+All the Design type buttons can be used as simple buttons.
 
-## Buttons with Icon
-All buttons styles can be used with an icon. You can use the `sap-icon--{icon-name}` class to attach an icon to the button.
-Full list of all the available icons can be found on the <a href="icon.html">icons page</a>.
-{% capture button %}
-<button class="fd-button fd-button--emphasized sap-icon--cart">Add to Cart</button>
-<button class="fd-button sap-icon--cart">Add to Cart</button>
-<button class="fd-button fd-button--light sap-icon--cart">Add to Cart</button>
-<button class="fd-button fd-button--positive sap-icon--accept">Approve</button>
-<button class="fd-button fd-button--negative sap-icon--decline">Reject</button>
-<br><br>
-<button class="fd-button sap-icon--cart"></button>
-<button class="fd-button fd-button--light sap-icon--cart"></button>
-<button class="fd-button fd-button--standard sap-icon--filter"></button>
-<button class="fd-button fd-button--positive sap-icon--accept"></button>
-<button class="fd-button fd-button--negative sap-icon--decline"></button>
-{% endcapture %}
-{% include display-component.html component=button %}
+### Toggle Button
 
+Use toggle button in a toolbar to activate/deactivate item or element. Another user case is to switch between different states.
```

Comment:
Use the toggle button in a toolbar to activate/deactivate an item or element. It can also be used to switch between different states.